Tasks/ServiceFabricDeploy/task.json

```diff
@@ -74,6 +74,14 @@
             "required": false,
             "helpMarkDown": "Path to the application parameters file. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and wildcards can be used in the path. If specified, this will override the value in the publish profile."
         },
+        {
+            "name": "overrideApplicationParameter",
+            "type": "boolean",
+            "label": "Override Application Parameters",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Variables defined in the Build or Release Definition will be matched against the 'key' or 'name' entries in the application manifest file. \n Example: If your application has a parameter defined as below. \n <Parameter Name=\"SampleApp_PartitionCount\" Value=\"1\" /> \n and you want to change the partition count to 2, you can define a release definition or an environment variable \"SampleApp_PartitionCount\" and its value as \"2\". \n\n Note: If same variables are defined in the Release Definition and in the Environment, then the Environment variables will supersede the Release Definition variables."
```

Comment:
Is the key and value present in the manifest file? Can you work with Pulkit to get the right message.